package.json

```diff
@@ -8,7 +8,7 @@
   "main": "build/index.js",
   "types": "build/",
   "scripts": {
-    "build": "yarn build:tsc && yarn build:build",
+    "build": "rm -rf build/ && yarn build:tsc && yarn build:build",
```

Comment:
So build will be a clean build each time? Should be more like a "clean" target?

Reply:
Yeah it could be a clean task as well. Really only run if the protos are existing in this directory and need to be re-generated